code/game/objects/items/weapons/soap.dm

```diff
@@ -35,9 +35,7 @@
 	reagents.add_reagent(/datum/reagent/cleaner, 15)
 
 /obj/item/soap/Crossed(var/mob/living/AM)
-	if (istype(AM))
```

Comment:
Is this (ctrl-click and drag someone over soap to slip them) a concern with grabs, considering the delays and downsides to establishing one?

Reply:
I think it's very different compared to quiet non-resistible pull.

Reply:
Yeah I'm gunna leave this removed.